js/escalade.js.php

```diff
@@ -28,7 +28,7 @@
 
       //set active group in red
       $("table:contains('$locale_actor') td:last, .tab_actors .actor-bloc:last")
-         .find("a[href*=group]")
+         .find("a[onclick*=submitGetLink]")
```

Comment:
I do not agree with this, as this link will not be present if user cannot assign ticket to himself. So it will just move issue to another case.

Reply:
done